### PR TITLE
Release Hermes v0.6.2

### PR DESCRIPTION
**Pull request by romac**
Closes: #1255

### CODE REVIEW

**Comment by romac** [comment]
This link check is failing but will pass once this is merged and we create the v0.6.2 tag:

> ERROR: 1 dead links found!
> [✖] https://github.com/informalsystems/ibc-rs/blob/v0.6.2/config.toml → Status: 404

File: .changelog/v0.6.2/summary.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+This minor release of Hermes re-enables the `upgrade client`,
```
Comment (adizere) [inline .changelog/v0.6.2/summary.md:1]:
```suggestion
This minor release of Hermes re-enables the `upgrade client`, `upgrade clients`,
```

**Review by adizere — COMMENTED** [review]
Before the release, something I missed in PR #1254 was to uncomment this code:

https://github.com/informalsystems/ibc-rs/blob/master/relayer-cli/src/commands/upgrade.rs#L16-L21

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,41 @@
 # CHANGELOG
 
+## v0.6.2
+
+This minor release of Hermes re-enables the `upgrade client`,
```
Comment (adizere) [inline CHANGELOG.md:5]: One last comment:

I see that the content of .changelog/v0.6.2/summary.md differs from this. Maybe we need to regenerate the changelog.

**Review by adizere — APPROVED** [review]

**Pull request merged**